Unit1/Module1.4/Mini-Project1.4/README.md

```diff
@@ -0,0 +1,6 @@
+# Mini Project 1.4:
```

Comment:
Regarding your question, Id' say lets either drop this mini-project or have them add an endpoint to their existing code from before, as you suggested. Id lean towards having them add something.